### PR TITLE
chore: release 0.51.0-rc.1 and fix libwallet build(nextnet)

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
Fixes lib wallet nextnet build

### CODE REVIEW

File: .github/workflows/build_libwallets_workflow.yml [hunk 1]
```diff
@@ -277,7 +277,7 @@ jobs:
               -headers "libwallet-ios-x86_64/libminotari_wallet_ffi.h" \
             -library "libwallet-ios-aarch64/libminotari_wallet_ffi.ios_aarch64.a" \
               -headers "libwallet-ios-aarch64/libminotari_wallet_ffi.h" \
-            -output libwallet-ios-xcframework/libtminoari_wallet_ffi_ios.xcframework
+            -output libwallet-ios-xcframework/libminoari_wallet_ffi_ios.xcframework
```
Comment (delta1) [inline .github/workflows/build_libwallets_workflow.yml]: lol is this still misspelled

Comment (SWvheerden) [reply .github/workflows/build_libwallets_workflow.yml]: I think you might be correct

File: changelog-nextnet.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.51.0-rc.1](https://github.com/tari-project/tari/compare/v0.51.0-rc.0...v0.51.0-rc.1) (2023-08-14)
```
Comment (brianp) [inline changelog-nextnet.md:5]: No content line needs explainer even for no-code change.

**Review by brianp — APPROVED** [review]
Need another explainer line for changelog otherwise LGTM

File: .github/workflows/build_libwallets_workflow.yml [hunk 3]
```diff
@@ -277,7 +277,7 @@ jobs:
               -headers "libwallet-ios-x86_64/libminotari_wallet_ffi.h" \
             -library "libwallet-ios-aarch64/libminotari_wallet_ffi.ios_aarch64.a" \
               -headers "libwallet-ios-aarch64/libminotari_wallet_ffi.h" \
-            -output libwallet-ios-xcframework/libminoari_wallet_ffi_ios.xcframework
+            -output libwallet-ios-xcframework/libmintoari_wallet_ffi_ios.xcframework
```
Comment (delta1) [inline .github/workflows/build_libwallets_workflow.yml]: LOL

**Pull request merged**